looper/utils.py

```diff
@@ -0,0 +1,157 @@
+""" Helpers without an obvious logical home. """
+
+from collections import defaultdict
+import logging
+import os
+import yaml
+
+
+_LOGGER = logging.getLogger(__name__)
+
+
+
+class CommandChecker(object):
```

Comment:
will have to think about where this ultimately belongs... CommandChecker will perhaps belong with PipelineInterface... in `pipeline_interface` we add a list of required commands, which will be read/checked by CommandChecker. Just a thought